README.md

```diff
@@ -35,6 +35,7 @@ A curated list of awesome mathematics resources.
         * [Ring Theory](#ring-theory)
         * [Galois Theory](#galois-theory)
         * [Lie Algebras](#lie-algebras)
+        * [Algebraic Statistics](#algebraic_statistics)
```

Comment:
could you please run this script in order to update the table of contents?
https://github.com/rossant/awesome-math/blob/master/build_toc.py